### PR TITLE
Improve documentation for Bracken ignore errorStrategy

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage/faq-troubleshooting.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
-## I get a warning during centrifuge_kreport process with exit status 255
+## I get a warning during CENTRIFUGE_KREPORT or KRAKENTOOLS_COMBINEKREPORTS_CENTRIFUGE process with exit status 255
 
 When a sample has insufficient hits for abundance estimation, the resulting `report.txt` file will be empty.
 
-When trying to convert this to a kraken-style report, the conversion tool will exit with a status code `255`, and provide a `WARN`.
+When trying to convert this to a kraken-style report or merging together, the tools will exit with a status code `255`, and provide a `WARN`.
 
-This is _not_ an error nor a failure of the pipeline, just your sample has no hits to the provided database when using centrifuge.
+This is _not_ an error nor a failure of the pipeline, just your sample has no hits to the provided database when using Centrifuge.
```
Comment (LilyAnderssonLee) [inline docs/usage/faq-troubleshooting.md:9]:
```suggestion
This is _not_ an error or a failure of the pipeline, just your sample has no hits in the provided database when using Centrifuge.
```

File: docs/usage/faq-troubleshooting.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+
+```nextflow
+process {
+    withName: BRACKEN_BRACKEN {
+        errorStrategy = 'retry'
+    }
+}
+```
+
+Other `errorStrategy` options can be found on the [Nextflow documentation](https://www.nextflow.io/docs/latest/process.html#errorstrategy).
```
Comment (LilyAnderssonLee) [inline docs/usage/faq-troubleshooting.md:31]: Wow, I just learned that I can set the error strategy for some of my meta-val processes. Never thought about it before.

**Review by LilyAnderssonLee — APPROVED** [review]
LGTM!

File: docs/usage/faq-troubleshooting.md (hunk 1 above)
Comment (jfy133) [reply docs/usage/faq-troubleshooting.md:9]: nor is the correct grammar :)

**Pull request merged**